packages/runtime/src/runtimes/react/components/MakeswiftComponent.tsx

```diff
@@ -34,9 +39,12 @@ export const MakeswiftComponent = memo(({ snapshot, label, type, description }:
 
   useRegisterDocument(rootDocument)
 
+  const builtinSuspense = useBuiltinSuspense(getRootElement(rootDocument).type)
+  const SuspenseOrFragment = builtinSuspense ? Suspense : Fragment
```

Comment:
I suspect the top-level `Suspense` in `MakeswiftComponent` is unnecessary, but I couldn't confirm it with certainty, so leaving it in conditionally for now.